### DISCUSSION TITLE
Not using expo and using install yarn with the new method (corepack)

### DISCUSSION THREAD

**Discussion by vricosti** [react-native-tvos/react-native-tvos#722] [Q&A] [answered]
Hello,

I'm inquiring whether it's feasible to utilize this project without Expo since my mobile application doesn't incorporate it. In the README.md, only commands with Expo are provided. Additionally, there's a recurring note in the README about installing yarn globally. However, my yarn has been installed through corepack, as now recommended. Will this still be compatible with the project?

Thank you.

**Comment by douglowder** [comment, marked as answer]
Yes you can use this project without Expo.

The section on starting a new project does recommend using the Expo CLI, but that does not mean that the mobile application will include Expo native modules. If you are more accustomed to use the community CLI (`@react-native-community/cli`), that should also work.

Yarn installed through corepack should work fine.
